README.md

```diff
@@ -36,4 +36,67 @@ The id is unique and changes with every new p2000 message.
 
 Extracting data can be done with a template like:
 
-`{{ state_attr('sensor.p2000_zwolle', 'melding') }}`
\ No newline at end of file
+`{{ state_attr('sensor.p2000_zwolle', 'melding') }}`
+
+----------------------------------------------------------------------------------
+### lovelace Dashboard
+
+You need markdown and logbook for the following settings
+
+More information on your dashboard.<br> tekst in Dutch.<br>
+![afbeelding](https://user-images.githubusercontent.com/62996429/144426649-70ec4732-14e1-4856-aa40-d8b81e99f3c3.png)
+
+#### markdown card
+```
+type: markdown
+content: >
+  Datum : {{ state_attr('sensor.p2000', 'datum' ) }}   Tijd: {{
+  state_attr('sensor.p2000', 'tijd') }}
+
+
+  Melding : {{ state_attr('sensor.p2000', 'melding') }}<br>
+
+
+  {{ state_attr('sensor.p2000', 'tekstmelding') }}<br>
+
+  -
+
+  Plaats: {{ state_attr('sensor.p2000', 'plaats') }}
+
+  Straat : {{ state_attr('sensor.p2000', 'straat') }} 
+
+  Regio : {{ state_attr('sensor.p2000', 'regio') }}
+
+  capcode : {{ state_attr('sensor.p2000', 'capstring') }}
+
+  lat : {{ state_attr('sensor.p2000', 'latitude') }}  long: {{
+  state_attr('sensor.p2000', 'longitude') }}
+
+  <br>
+
+  Dienst: {{ state_attr('sensor.p2000', 'dienst') }}
+
+  info :  {{ state_attr('sensor.p2000', 'brandinfo') }}<br>
+
+  Regio id : {{ state_attr('sensor.p2000', 'regioid') }}       Dienst id : {{
+  state_attr('sensor.p2000', 'dienstid') }}
+
+  Prio : {{ state_attr('sensor.p2000', 'prio1') }}
+
+  Grip : {{ state_attr('sensor.p2000', 'grip') }}
+
+
+  Id nr : {{ state_attr('sensor.p2000', 'id') }}<br>
+```
+----------------------------------------------------------------------------------
+Also <br>
+![afbeelding](https://user-images.githubusercontent.com/62996429/144427222-1118abfa-4710-4f48-962f-b9f89cd523bc.png)
```

Comment:
The images will be missing if merged; Please add them to the assets folder and use them like
`(./assets/screenshot01.png)`